web/src/settings.js

```diff
@@ -105,6 +105,11 @@ export function update_lock_icon_in_sidebar() {
 export function build_page() {
     setup_settings_label();
 
+    const settings_is_disabled = {
+        message_content_in_email_notifications:
+            !page_params.realm_message_content_allowed_in_email_notifications,
+    };
```

Comment:
I think you can directly pass `message_content_in_email_notifications_disabled` as an argument below.

Reply:
fixed in the latest commit

Reply:
I meant here that you can have `message_content_in_email_notifications_disabled: !page_params.realm_message_content_allowed_in_email_notifications` directly in the args passed to `render_settings_tab`.

Reply:
Ok, I was thinking of something else, but then I noticed that we use a loop to render the email notification settings. Hmm, I am not sure this is the right way to do this. We probably need to refactor some code in `settings_config.ts` that handles all these notification setttings.

We have `show_push_notifications_tooltip` which gives the status of whether the setting is disabled or not, but that is currently used only for mobile push notification related settings. We can probably rename it to something like `disabled_notification_settings` and then add `message_content_allowed_in_email_notifications` in that object.

Reply:
A temporary workaround could be to disable the settings in JS rather in doing it in handlebar templates, but I guess the above approach seems good for long-term. Tbh I was confused why we use `show_push_notifications_tooltip` object to check whether the setting is enabled or not. @timabbott your thoughts on this?